activedirectory/azure-ad-integration-spring-boot-autoconfigure/pom.xml

```diff
@@ -40,6 +40,10 @@
         <url>https://github.com/Microsoft/azure-spring-boot-starters/tree/master</url>
     </scm>
 
+    <properties>
+        <azure.adal4j.version>1.2.0</azure.adal4j.version>
+    </properties>
```

Comment:
make the version of Azure package a property so that it can be overwritten

Reply:
@ZhijunZhao ordering dependencies alphabetically can improve readability and also adding new dependencies later. if you agree I will propose a PR

Reply:
That would be fantastic! But it would be better after this PR is merged, which should be early tomorrow morning.

Reply:
I will submit in next couple of days once these spring boot initializr things completes.